frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1179,37 +1186,46 @@ function ContextMenuInner(propsWithInputRef: Props) {
   } = props;
 
   const segmentIdAtPosition = globalPosition != null ? getSegmentIdForPosition(globalPosition) : 0;
-  const hasNoFallbackLayer =
-    visibleSegmentationLayer != null && !hasFallbackLayer(visibleSegmentationLayer);
+
+  // Currently either segmentIdAtPosition or maybeClickedMeshId is set, but not both.
+  // segmentIdAtPosition is only set if a segment is hovered in one of the xy, xz, or yz viewports.
+  // maybeClickedMeshId is only set, when a mesh is hovered in the 3d viewport.
+  // Thus the segment id is always unambiguous / clearly defined.
+  const clickedSegmentOrMeshId =
+    maybeClickedMeshId != null ? maybeClickedMeshId : segmentIdAtPosition;
+  const isHoveredSegmentOrMesh = clickedSegmentOrMeshId > 0;
+
+  const { dataset, tracing, flycam } = useSelector((state: OxalisState) => state);
+  useEffect(() => {
+    Store.dispatch(ensureSegmentIndexIsLoadedAction(visibleSegmentationLayer?.name));
+  }, [visibleSegmentationLayer]);
+  const isSegmentIndexAvailable = useSelector((state: OxalisState) =>
+    getMaybeSegmentIndexAvailability(state.dataset, visibleSegmentationLayer?.name),
+  );
+  const isLoadingVolumeAndBB = ["loading", "loading"];
   const [segmentVolume, boundingBoxInfo] = useFetch(
```

Comment:
I'd rename these vars to `segmentVolumeLabel` and `boundingBoxInfoLabel`. That way, it is a bit more expected that the labels can contain strings, such as "loading" or "could not be fetched".